glhe/aggregation/dynamic.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
The fancy `|` operator for annotating optional types is only available in Python 3.10 onward. I have 3.9 on my system, so this is needed for us lower life forms.